### PR TITLE
boot-script recipe for easily demoing OTA updates

### PR DESCRIPTION
**Pull request by estenberg**
can be used to blink lights, show picture, video, etc.

### CODE REVIEW

**Comment by estenberg** [comment]
Tested with blinking lights demo on minnowboard, but can be useful for most demo cases.

Is there a better way to "include" it than in the file core-image-full-cmdline.bbappend?
Would be nice if it was not specific to bitbake build target.

File: meta-mender-demo/recipes-mender/boot-script/boot-script.bb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+LICENSE = "Apache-2.0"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI = "file://boot-script.service file://boot-script.sh file://LICENSE"
+
```
Comment (kacf) [inline meta-mender-demo/recipes-mender/boot-script/boot-script.bb]: Try to insert

```
S = "${WORKDIR}"
```

I think that will get rid of the `LICENSE` error you're getting in the Jenkins build.

File: meta-mender-demo/recipes-core/images/core-image-full-cmdline.bbappend [hunk 2]
```diff
@@ -1 +1 @@
-IMAGE_INSTALL_append = " hello-mender"
+IMAGE_INSTALL_append = " hello-mender boot-script"
```
Comment (kacf) [inline meta-mender-demo/recipes-core/images/core-image-full-cmdline.bbappend]: > Is there a better way to "include" it than in the file core-image-full-cmdline.bbappend?
> Would be nice if it was not specific to bitbake build target.

Yes, I believe so. In fact, I think we should move both, so open the `meta-mender-demo/recipes-core/packagegroups/packagegroup-core-boot.bbappend`, and add it like this:

```
RDEPENDS_${PN}_append = " hello-mender boot-script"
```

and then just delete the `core-image-full-cmdline.bbappend` file.

**Comment by estenberg** [comment]
Thanks @kacf ! Updated.

File: meta-mender-demo/recipes-mender/boot-script/boot-script.bb (hunk 1 above)
Comment (estenberg) [reply meta-mender-demo/recipes-mender/boot-script/boot-script.bb]: The secret way to get Jenkins to be happy? I like it ;)

Comment (kacf) [reply meta-mender-demo/recipes-mender/boot-script/boot-script.bb]: Hehe, yes, it's a long and complicated reason, so you just have to trust me that it *does* make sense! :-)

**Comment by kacf** [comment]
The usual spurious failures. Those have become really bad lately...

**Comment by kacf** [comment]
I will cherry-pick this later as part of a bigger sumo update.

**Pull request merged**